README.md

```diff
@@ -37,6 +37,8 @@ Managing state can become difficult to coordinate when an application becomes co
 
 Dojo 2 stores provides a centralized store, designed to be the **single source of truth** for an application. It operates using uni-directional data flow. This means all application data follows the same lifecycle, ensuring the application logic is predictable and easy to understand.
 
+__Note__: Using and passing local state around without using a centralized application store like @dojo/stores is an acceptable architectural pattern.
```

Comment:
Perhaps re-phrase this in a why that it is a question? Maybe something similar to this?

> "Do you need a centralised store? Lifting state up to parent widgets and using local state are likely to be sufficient in less complex applications"

Reply:
Nicer 👍  changed